client/src/components/ProductDetails/StyleSelector.jsx

```diff
@@ -14,14 +11,18 @@ function StyleSelector ({productStylesArray, currentStyleID, changeCurrentStyle}
 
   const currentStyleObject = getCurrentStyleObject(currentStyleID, productStylesArray);
 
+
   return (
     <div className="flex-container wrap">
-      <h2>{currentStyleObject[0]['name']}</h2>
-      <h4>{`$${currentStyleObject[0]['original_price']}`}</h4>
+      <h5>{currentProduct.category}</h5>
+      <h2>{currentProduct.name}</h2>
+      <h5>${currentStyleObject[0]['original_price']}</h5>
```

Comment:
I prefer working with objects, because the keys can help make the code more readable -- but that's just a personal preference.